### ISSUE TITLE
Getting error when i use any "bower" command after successful installation of bower 

### ISSUE THREAD

**Issue by Chetan0793** [angular/angular-phonecat#321]
/usr/local/lib/node_modules/bower/lib/node_modules/configstore/index.js:56
				throw err;
				^

Error: EACCES: permission denied, open '/Users/CRK/.config/configstore/bower-github.yml'
You don't have access to this file.

    at Error (native)
    at Object.fs.openSync (evalmachine.<anonymous>:584:18)
    at Object.fs.readFileSync (evalmachine.<anonymous>:431:33)
    at Object.create.all.get (/usr/local/lib/node_modules/bower/lib/node_modules/configstore/index.js:34:29)
    at Object.Configstore (/usr/local/lib/node_modules/bower/lib/node_modules/configstore/index.js:27:44)
    at readCachedConfig (/usr/local/lib/node_modules/bower/lib/config.js:19:23)
    at defaultConfig (/usr/local/lib/node_modules/bower/lib/config.js:11:12)
    at Object.<anonymous> (/usr/local/lib/node_modules/bower/lib/index.js:16:32)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)

**Comment by wrjanan** [comment]
Hey, I do not know if it will help you, I solved my problem by listing out the bower changes.

bower list

I realised that there were some inconsistencies, and remembered that I manually edited some values such that the version was way ahead of what the module actually had. 

Example:
```
    "angular-resource": "^1.6.0",
    "angular-cookies": "^1.6.0",
    "angular-sanitise": "^1.6.0",
```

Angular is only at 1.5.5 and I changed it to 1.6.0. LOL!!! FOOLISH ME~
change to:
```
    "angular-resource": "^1.4.0",
    "angular-cookies": "^1.4.0",
    "angular-sanitise": "^1.4.0",
```

after changing the dependency that you think is contentious from the "bower-list"
do a "bower update -a"

let me know if this helps ya~

It worked for me after searching around, the main problem is manually editing bower.json.
If we are using bower cli, we should let it always manage our packages.

Do not worry, know once and forever remember :)
